src/AutoBuild.jl

```diff
@@ -538,9 +538,9 @@ function register_jll(name, build_version, dependencies, julia_compat;
         upstream_registry_url = "https://github.com/JuliaRegistries/General"
         name_jll = "$(name)_jll"
         if _package_is_registered(upstream_registry_url, name_jll)
-            pr_title = "New version: $(name_jll) v$(build_version)"
+            pr_title = "[$(name)] New version: $(name_jll) v$(build_version)"
```

Comment:
The name is already there, no?

Reply:
```suggestion
            pr_title = "[$(name)] New version: v$(build_version)"            
```

Yes, just not at beginning of title, would you remove it from the second part like this?

Reply:
I mean, I don't understand the need to change this just for the sake of changing? What problem is this meant to solve?

Reply:
The PR is about taking a syntax / pattern which has become the (manual) dominant pattern and making it the automatic default pattern. It's a non-technical change and one which is a minor tweak, so feel free to close this PR, but I was checking out the BinaryBuilder codebase and it was easy to see how it might be implemented. Here are a few screen shots to compare; perhaps it would be good to get further community feedback before implementing though?
<img width="182" alt="Screen Shot 2021-12-14 at 16 07 06" src="https://user-images.githubusercontent.com/4462211/146087218-bad39474-2bdc-41f8-9245-d5eeb43b4545.png">

<img width="1247" alt="Screen Shot 2021-12-14 at 16 06 49" src="https://user-images.githubusercontent.com/4462211/146087224-e34f5644-b1b0-4c2f-b445-d3034cfbedea.png">
<img width="1233" alt="Screen Shot 2021-12-14 at 16 06 21" src="https://user-images.githubusercontent.com/4462211/146087240-a80ed4c6-fb2a-4e11-9b16-6cc6fef67702.png">

<img width="1261" alt="Screen Shot 2021-12-14 at 16 06 37" src="https://user-images.githubusercontent.com/4462211/146087228-8f07411a-e460-4d4b-ab01-33930ab3d872.png">